tests/testthat/test-markdown.R

```diff
@@ -1,3 +1,10 @@
 test_that("empty string works", {
   expect_equal(markdown_text(""), "")
 })
+
+test_that("header attributes are parsed", {
+  index_xml <- markdown_text(read_lines(test_path("assets/readme-with-attr.md")))
```

Comment:
I meant just inline the file, i.e.:

```R
markdown_text("# Header {.class #id}")
```

Reply:
Ah, of course. Just pushed changes and removed the now unnecessary `tests/testthat/assets/reamd-with-attr.md` file as well.